pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -130,9 +136,10 @@ private void initializeApplicationQpsQuotas() {
 
       String appName = entry.getKey();
       double appQpsQuota =
-          entry.getValue() != null && entry.getValue() != -1.0d ? entry.getValue() : _defaultQpsQuotaForApplication;
+          entry.getValue() != null && entry.getValue() >= MIN_APP_QUOTA ? entry.getValue()
```

Comment:
Not introduced in this PR, but we might want to allow overriding default quota to disable throttling
```suggestion
          entry.getValue() != null ? entry.getValue() : _defaultQpsQuotaForApplication;
```

Reply:
Applied.